qiskit/primitives/sampler.py

```diff
@@ -82,6 +82,22 @@ def __init__(
         super().__init__(preprocessed_circuits, parameters, options)
         self._is_closed = False
 
+    def __repr__(self):
+        circ_names = []
+        for circuit in self._circuits:
+            circ_names.append(circuit.name)
+        param_counts = []
+        for param in self._parameters:
+            param_counts.append(str(len(param)))
+
+        return "< {} with {} circuits ('{}',) parameter counts ({},) and {} >".format(
```

Comment:
Sorry for the same comment.

1. extra space around `<` and `>`
2. Passing circuits during initialization is now deprecated. Therefore, it is better not to display them here as well and just display `_run_options`.